Deploy/kubernetes/ui/ui.rc.yaml

```diff
@@ -0,0 +1,169 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: ui-rc
+  labels:
+    name: ui-rc
+spec:
+  replicas: 1
+  selector:
+    name: ui-apps
+  template:
+    metadata:
+      labels:
+        name: ui-apps
+    spec:
+      containers:
+      - name: ui-app
+        image: harbor/ui
```

Comment:
Image is hardcoded here?  This doesn't make sense as you've mentioned in the README that we'll use versioned ones, right?